Render the plugin in a custom render hook

abishekrsrikaanth opened the pull request:
Adding functionality to render the plugin in a  custom render hook. This is useful when a developer customizes their theme and the plugin creates UI issues when it renders using the default hook.
Updated the documentation for the changes.

awcodes commented:
What does this even look like in other places? It's kinda designed to be in the top bar only.

abishekrsrikaanth commented:
This gives flexibility to pretty much move the plugin anywhere on the panel. For the current app, I publish the panel view assets and setup my own render hooks and here is an example of a custom hook I have added after the logo. For this app, I have removed the top bar and only have the left sidebar. And here is how the sidebar looks as well 

![CleanShot 2024-05-28 at 15 30 38](https://github.com/awcodes/filament-quick-create/assets/1639302/8f9dee54-5d73-4f4d-8b35-b91829957d1c)

![CleanShot 2024-05-28 at 15 37 12](https://github.com/awcodes/filament-quick-create/assets/1639302/7aad7ee9-7e14-40a8-9f68-f708d7d4123f)

And if the developer needed, they can publish the quick create view and customize the view and move it to a suitable location of their choice. I hope that clarifies.
